README.md

```diff
@@ -147,8 +154,6 @@ jobs:
       - id: do-stuff
         uses: grafana/shared-workflows/actions/my-new-action@my-new-action-v1.0.0
 ```
-
-<!-- x-release-please-end-version -->
 ````
 
-Every semver-like string between the `x-release-please-start-version` and `x-release-please-end-version` comments will be updated with the new version whenever the component is released.
+<!-- x-release-please-end-version -->
```

Comment:
This should stay in the visible bit no?

Reply:
Oh docs got messed up. Yes, gonna revert that change